src/style.css

```diff
@@ -1 +1,51 @@
 /* add styles here */
+
+.page {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
```

Comment:
You have repeated styles for background-repeat and background-position in the .stars__star class and also in the .stars--1 .stars__star class. To adhere to the DRY (Don't Repeat Yourself) principle, consider removing the repeated properties from the .stars--1 .stars__star class (lines 32-34) as they are already defined in the .stars__star class.